examples/Vanilla/InstrumentationVersionCollector/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8" />
+    <title>Instrumentation Version Collector Example</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <h1>Instrumentation Version Collector</h1>
+    <p>Please check the network tab to inspect the event. It should have softwareInfo 5 field populated.</p>
+    <button autofocus onclick="window.location.href='../index.html'">
+      Home
+    </button>
+    <button autofocus onclick="window.location.href='../Error/index.html'">
+      Back
+    </button>
+    <button autofocus onclick="window.location.href='../OnLoad/index.html'">
```

Comment:
@robertdumitrescu [BUG] copy-paste error - remember to update link URL so that prev/next buttons are chaining all examples one by one, also, update index.html on the root directory to contain this new example

Reply:
OK - fixed